src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -1768,7 +1768,7 @@ <h2 class="text--center">Balance sheet</h2>
 
               {{if $debtors := $currentAssets.debtors}}
                 <tr class="figures">
-                  <td id="ac52-label">Debtors:</td>
+                  <td id="debtors-label">Debtors:</td>
```

Comment:
Do we need to match our ids to the ids in the provided SmallFullAccountsModifiediXBRL.html?